models/Element/Service.php

```diff
@@ -591,6 +591,12 @@ public static function getElementHash(ElementInterface $element): ?string
      */
     public static function getType($element)
     {
+        trigger_deprecation(
+            'pimcore/pimcore',
+            '10.0',
```

Comment:
Shouldn't this be `10.2.0`? 
Docs say: `@param string $version The version of the package that introduced the deprecation`

Reply:
Is deprecated since 10.0

Reply:
yep, just `trigger_deprecation()` was missing 😉